app/lib/common/providers/notifiers/chat_notifiers.dart

```diff
@@ -70,20 +70,15 @@ class AsyncLatestMsgNotifier extends FamilyAsyncNotifier<RoomMessage?, String> {
   }
 }
 
-class ChatRoomsListNotifier extends StateNotifier<List<Convo>> {
-  final Client client;
+class ChatRoomsListNotifier extends Notifier<List<Convo>> {
```

Comment:
Notifier already has `ref` given and without `Client` passed in I mad most `StateNotifiers` to simple `Notifiers` instead.